### PR TITLE
meta(codeowners): Split some visibility codeownership

### PR DESCRIPTION
**Pull request by wmak**
- Just a first pass of moving some files to the individual performance &
  discover-dashboards teams.
- Also moving issue search to workflow

### CODE REVIEW

**Comment by billyvg** [comment]
Is "visibility" now an umbrella for perf + dnd?

**Comment by wmak** [comment]
:+1: Yep, for now at least, there's going to still be a decent amount of overlap between the two teams

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -104,62 +104,66 @@ build-utils/        @getsentry/owners-js-build
...
-/static/app/views/dashboardsV2/                                 @getsentry/visibility
-/tests/js/spec/views/dashboardsV2/                              @getsentry/visibility
-/static/app/components/events/interfaces/spans/                 @getsentry/visibility
-/static/app/components/performance/                             @getsentry/visibility
-/static/app/components/quickTrace/                              @getsentry/visibility
-### Endof Perfomance/Discover ###
+### Endof Visibility ###
+
+
+### Perfomance ###
```
Comment (dashed) [inline .github/CODEOWNERS:126]:
```suggestion
### Performance ###
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -104,62 +104,66 @@ build-utils/        @getsentry/owners-js-build
...
+/src/sentry/api/endpoints/organization_events_vitals.py         @getsentry/performance
+
+/static/app/views/performance/                                  @getsentry/performance
+/tests/js/spec/views/performance/                               @getsentry/performance
+/static/app/utils/performance/                                  @getsentry/performance
+/tests/js/spec/utils/performance/                               @getsentry/performance
+/static/app/components/events/interfaces/spans/                 @getsentry/performance
+/static/app/components/performance/                             @getsentry/performance
+/static/app/components/quickTrace/                              @getsentry/performance
+### Endof Perfomance ###
```
Comment (dashed) [inline .github/CODEOWNERS:140]:
```suggestion
### Endof Performance ###
```

**Review by dashed — APPROVED** [review]

**Pull request merged**